VimeoUpload/Upload/Model/VideoSettings.swift

```diff
@@ -36,7 +36,8 @@ class VideoSettings: NSObject
 
     init(title: String?, description: String?, privacy: String?, users: [String]?, password: String?)
     {
-        self.title = title
+        // Remove trailing whitespace [CL]
+        self.title = title?.trim()
```

Comment:
Rather than putting this in init can we override didSet? That way if we modify the title in somewhere other than initialization it will be trimmed as well like 

`
        didSet
        {
            self.title = title?.trim()
        }
`